src/lightning_app/core/api.py

```diff
@@ -23,7 +23,13 @@
 
 from lightning_app.api.http_methods import HttpMethod
 from lightning_app.api.request_types import DeltaRequest
-from lightning_app.core.constants import CLOUD_QUEUE_TYPE, ENABLE_STATE_WEBSOCKET, FRONTEND_DIR
+from lightning_app.core.constants import (
+    CLOUD_QUEUE_TYPE,
+    ENABLE_PULLING_STATE_ENDPOINT,
+    ENABLE_PUSHING_STATE_ENDPOINT,
+    ENABLE_STATE_WEBSOCKET,
```

Comment:
For the future, do you think we should leverage the `.lightning` file more to have all app-related configurations there? env variables should remain usable be can be overridden by app_config.

Reply:
I actually think that some of these should eventually be part of the API (for instance as arguments to the App constructor, or arguments of the flow). Exposing the state or not is a concern of the app developer, rather than the one who runs the app IMO.

Reply:
We can consider it later. But let's keep with env variables. It is more reversive than new arguments.

Reply:
Yes for sure, my comment was about future developments